core/src/main/java/dev/keva/core/config/KevaConfig.java

```diff
@@ -16,12 +16,12 @@
 @AllArgsConstructor
 @Configuration
 public class KevaConfig {
-    @ConfigProp(name = "hostname", defaultVal = "localhost")
-    @CliProp(name = "h", type = CliPropType.VAL)
+    @ConfigProp(name = "host", defaultVal = "localhost")
+    @CliProp(name = "host", type = CliPropType.VAL)
     private String hostname;
 
     @ConfigProp(name = "port", defaultVal = "6379")
-    @CliProp(name = "p", type = CliPropType.VAL)
+    @CliProp(name = "port", type = CliPropType.VAL)
```

Comment:
but why :cry:

Reply:
We will try to follow Redis Conf standard

Reply:
Check this https://redis.io/topics/config

Reply:
```
./redis-server --port 6380 --replicaof 127.0.0.1 6379
```

They don't have `-p` option either

Reply:
ok i c